docs/quickstarts/aws.md

```diff
@@ -163,6 +163,19 @@ export KUBECONFIG=./lokomotive-assets/cluster-assets/auth/kubeconfig
 kubectl get nodes
 ```
 
+## Using the cluster
```

Comment:
Should we have some general quick start guide and leave platform specific documents for only platform specific information to avoid such duplication? I feel, that it could quickly get out of sync.

Reply:
Yes, can you create an issue about that?

Reply:
Done: #234.